lib/agent0/agent0/interactive_fuzz/helpers/open_random_trades.py

```diff
@@ -56,6 +56,9 @@ def open_random_trades(
         if advance_time:
             # Advance a random amount of time between opening trades
             chain.advance_time(
-                rng.integers(low=0, high=interactive_hyperdrive.hyperdrive_interface.pool_config.position_duration)
+                rng.integers(
+                    low=0,
+                    high=interactive_hyperdrive.hyperdrive_interface.pool_config.position_duration,
+                )
```

Comment:
We should probably set create_checkpoint=True here, right? It's getting used in a few places so I'm not sure, but I think we might be in trouble if we advance past a checkpoint interval and don't make one.

Reply:
huh?

Reply:
opening a trade creates a checkpoint by default